frontend/src/queries/schema.json

```diff
@@ -19421,6 +19421,10 @@
             ],
             "type": "string"
         },
+        "RecordingOrderDirection": {
```

Comment:
for @TueHaulund's context

we change the schema.ts file
and then run `pnpm schema:build`
and that generates a front and back end schema definition